weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableColumns.tsx

```diff
@@ -471,7 +471,7 @@ function buildCallsTableColumns(
   cols.push(startedAtCol);
 
   cols.push({
-    field: 'derived.tokens',
+    field: 'summary.weave.tokens',
```

Comment:
i am assuming this is for forwards compat? doesn't really effect things yet, correct?